kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -270,29 +270,15 @@ define([
             }));
 
             stagingAreaViewer.$elem.append($fileTable);
-            stagingAreaViewer.$elem.find('table').dataTable({
+            
+            const fullDataTable = stagingAreaViewer.$elem.find('table').dataTable({
```

Comment:
this is the only way to get the full data from the table vs. what is just rendered in the current DOM